packages/fxa-auth-server/fxa-oauth-server/config/dev.json

```diff
@@ -176,13 +177,31 @@
       "privacyUri": "",
       "trusted": true,
       "publicClient": false
+    },
+    {
+      "id": "a8c528140153d1c6",
+      "name": "Firefox Secure Proxy",
+      "hashedSecret": "0000000000000000000000000000000000000000000000000000000000000000",
+      "redirectUri": "https://cb7cbf5bedba243279adcd23bc6b88de7a304388.extensions.allizom.org/",
+      "imageUri": "",
+      "canGrant": true,
+      "trusted": true,
+      "publicClient": true
     }
   ],
   "localRedirects": true,
   "logging": {
     "level": "ALL",
     "fmt": "pretty"
   },
+  "jwtAccessTokens": {
+    "enabled": true,
+    "enabledClientIds": [
+      "98e6508e88680e1a",
```

Comment:
Comment these are for fxa-settings, 123done, and secure proxy. Though remove secure proxy and add that to stomlinson.dev.lcip.org config.

Reply:
Why fxa-settings, is it just so we can test it out ourselves concurrently with the proxy team's work?

Reply:
> Why fxa-settings, is it just so we can test it out ourselves concurrently with the proxy team's work?

Exactly. And because settings sends access tokens to the auth server whereas the others work directly with the oauth server. Sending access tokens to the auth server showed that I hadn't updated the validator logic there to handle JWT access tokens.